### PR TITLE
Break animation

### LINKED ISSUE #59: Implement breaking animation
at least have the same animation as vanilla
currently from an user perspective, it's not clear whether it works

* [ ] breaking animation

### LINKED ISSUE #121: Biggest asks
Listing here the current biggest asks from users:
* [ ] #117 ctrl+w closing the window is confusing for 50% of new users
* [ ] #28 mouse sensitivity is a recurrent ask
* [ ] #59 breaking animation being absent is confusing many users: seems like digging doesn't work
* [ ] #10 inventory implementation seems needed

### PR DESCRIPTION
**Pull request by AwesomestCode**
Closes #59
Closes #121

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -30,6 +30,20 @@ document.body.appendChild(renderer.domElement)
...
 
+const textures = []
+
+const loader = new THREE.TextureLoader()
+
+let breakStartTime
+
+for (let i = 0; i < 10; i++) {
+  loader.load('textures/1.16.4/blocks/destroy_stage_' + i + '.png', (texture) => {
+    texture.magFilter = THREE.NearestFilter
```
Comment (rom1504) [inline index.js:41]: Don't hardcode, get the version from viewer.version

**Review by rom1504 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Karang) [reply index.js]: Version isn't loaded at this stage. To get the version you need to move this loop into the spawn event handler.

**Comment by rom1504** [comment]
Thanks this looks reasonable.

I'll try it later, if it works well let's merge

(I agree the feature to have pr auto deployed to gh pages would be helpful)

File: index.js (hunk 1 above)
Comment (AwesomestCode) [reply index.js]: Karang: I moved it out.

**Pull request merged**